todoman/model.py

```diff
@@ -603,6 +605,14 @@ def todos(self, all=False, lists=[], urgent=False, location='',
             max_due = (datetime.now() + timedelta(hours=due)).timestamp()
             extra_where.append('AND due IS NOT NULL AND due < ?')
             params.append(max_due)
+        if start:
+            if before:
+                extra_where.append('AND created_at <= ?')
+                params.append(start)
+            else:
+                extra_where.append('AND created_at >= ?')
+                params.append(start)
```

Comment:
No, not at all. I didn't know that one can do that! I opened #125 for further investigation.